skipper.go

```diff
@@ -720,6 +724,7 @@ func createDataClients(o Options, auth innkeeper.Authentication) ([]routing.Data
 			KubernetesEastWestDomain:   o.KubernetesEastWestDomain,
 			DefaultFiltersDir:          o.DefaultFiltersDir,
 			OriginMarker:               o.EnableRouteCreationMetrics,
+			BackendNameTracingTag:      o.OpenTracingBackendNameTag,
```

Comment:
Why not DefaultFilters PreProcessor as in https://github.com/zalando/skipper/blob/master/skipper.go#L1179